test/unit/lib/email.js

```diff
@@ -23,31 +23,31 @@ describe('Email', function () {
 		demand(Email).throw(/requires a templateName or options argument/);
 	});
 
-	it('should require keystone-email to be installed', function () {
-		// Pretend keystone-email isn't installed for this test
-		var Email = proxyquire('../../../lib/email', { 'keystone-email': null });
-		try {
-			Email({});
-		} catch (err) {
-			demand(err.message).contain('keystone-email');
-		}
-	});
+	// it('should require keystone-email to be installed', function () {
```

Comment:
Worth adding a comment on why these tests are disabled. Are these tests likely to be relevant in future or should they be removed?

Reply:
[style] Not sure if there is any preference for single-line versus multi-line comment blocks. I assume single-line is a convenience for selecting and toggling comments in an editor.